automation/devops_automation_infra/installers/k8s.py

```diff
@@ -37,7 +37,8 @@ def clean(cluster, request):
     logging.info("running devops clean_base_btwn_tests")
     cluster.ProxyDaemonSet.restart()
     for host in cluster.hosts.values():
-        waiter.wait_nothrow(host.SSH.connect, timeout=30)
+        if host.user != 'oosto':
```

Comment:
Maybe worth to do the user `oosto` a const value and use it in other places (if needed)

Reply:
this is temporarily way to detect the jetson node until we do it using architecture filed like x86/arm

Reply:
Sounds good 👍